aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -123,6 +125,29 @@ spec:
     - p3.8xlarge
     - p3.16xlarge
 
+  - name: AddIAMUser
+    description: "Do you want to add extra user to have an access to EKS via kubectl"
+    type: Confirm
+    default: true
+
+  - name: EKSUserARN
+    description: "Enter existent IAM user ARN to be used to access EKS cluster via kubectl"
+    type: Input
+    default: "arn:aws:iam::100000000000:user/username"
```

Comment:
Is this a valid ARN?

Reply:
nope just example. can be put on description but then will be too long